examples/security/secrets/secret-setup/src/main.rs

```diff
@@ -0,0 +1,50 @@
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // put secrets in secret store
+    let component_secret = wasmcloud_secrets_types::Secret {
+        name: "api_password".to_string(),
+        string_secret: Some("opensesame".to_string()),
+        ..Default::default()
+    };
+    let provider_secret = wasmcloud_secrets_types::Secret {
+        name: "redis_password".to_string(),
+        string_secret: Some("sup3rS3cr3tP4ssw0rd".to_string()),
+        ..Default::default()
+    };
+
+    let nats_client = async_nats::connect("127.0.0.1:4222").await?;
+
+    put_secret(&nats_client, component_secret).await?;
+    put_secret(&nats_client, provider_secret).await?;
+
+    Ok(())
+}
+
+async fn put_secret(
```

Comment:
NIT: This feels like it could go on a trait in `wasmcloud_secrets_types` and be implemented for `async_nats::Client` under a feature flag or something... Sure wouldn't want to write this again

Reply:
Yeah I think this should be a part of a library from the nats-kv implementation of the secrets backend, since this is specifically built for putting a secret there

Reply:
Addressed by moving the function to the NATS KV library